types/testing-library__react/index.d.ts

```diff
@@ -9,6 +9,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
+import { OptionsReceived as PrettyFormatOptions } from 'pretty-format';
```

Comment:
Hey Kent :smile: I'm guessing that the build failed because pretty-format doesn't exist in this package. `testing-library__react` doesn't have a package.json with that dependency yet